src/eos-crash-metrics.c

```diff
@@ -0,0 +1,181 @@
+/* Copyright 2017 Endless Mobile, Inc. */
+
+#include <stdlib.h>
+#include <eosmetrics/eosmetrics.h>
+#include <ostree.h>
+
+#define PROGRAM_DUMPED_CORE_EVENT "ed57b607-4a56-47f1-b1e4-5dc3e74335ec"
+
+static GVariant *
+hashtable_to_variant (GHashTable *ht)
```

Comment:
Instead of this function, and the code below to populate the hash table, I think it would be much easier to use `GVariantDict`. You would initialize one on the stack, call `g_variant_dict_init()` and then call `g_variant_dict_insert()` or `g_variant_dict_insert_value()` to populate it. Then `g_variant_dict_end()` would give you the `a{sv}` variant.

Reply:
Oh handy. Okay, lets use that then.

Reply:
Done

Reply:
This function is unused now and should be removed.